bitwardenrs/templates/deployment.yaml

```diff
@@ -193,6 +193,8 @@ spec:
         {{- if .Values.persistence.enabled }}
         persistentVolumeClaim:
           claimName: {{ if .Values.persistence.existingClaim }}{{ .Values.persistence.existingClaim | quote }}{{- else }}{{ include "bitwardenrs.fullname" . }}{{- end }}
+        {{- else if .Values.customVolumes.enabled }}
+{{- toYaml .Values.customVolumes.config | nindent 8 }}
```

Comment:
Small nitpick, this breaks the indentation style of the file

Reply:
Indeed. To be even more pedantic: if it is a single volume, configuration should be "customVolume" ;-)

Reply:
will fix both